setup.py

```diff
@@ -41,12 +41,13 @@
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
     install_requires=[
-        "pytket ~= 1.3",
+        "pytket ~= 1.4",
         "requests >= 2.2",
         "types-requests",
         "websockets >= 7.0",
         "nest_asyncio >= 1.2",
         "pyjwt ~= 2.4",
+        "msal ~= 1.18",
```

Comment:
What is this for?

Reply:
It is for the Microsoft login. (added with the other MFA stuff)